docs/examples/metrics/example.py

```diff
@@ -1,16 +1,16 @@
 from typing import Iterable
 
-from opentelemetry._metrics import (
+from opentelemetry.exporter.otlp.proto.grpc._metric_exporter import (
```

Comment:
This `_metric_exporter` needs to be public too?

Reply:
Fixed

Reply:
This is still not updated?